src/amber/cli/templates/auth.cr

```diff
@@ -25,7 +25,7 @@ module Amber::CLI
       @timestamp = Time.now.to_s("%Y%m%d%H%M%S")
       @primary_key = primary_key
       @visible_fields = @fields.reject(&.hidden).map(&.name)
```

Comment:
we should remove this if its no longer being used.

Reply:
**@visible_fields** is still being use in other places

Reply:
@visible_fields is being used everywhere forms or controllers are being generated.